docs/discovery.md

```diff
@@ -128,6 +128,7 @@ https://platform.example/organization/123/presentations/prove
 https://platform.example/organization/123/presentations/verify
 https://platform.example/organization/123/presentations/available
 https://platform.example/organization/123/presentations/submissions
+https://platform.example/organization/123/presentations
```

Comment:
intention is to support OAuth on this and exchange VPs that don't contain a proof?

Reply:
Correct, that is the intention fo this